pkg/models/manifest.go

```diff
@@ -44,6 +44,7 @@ const (
 	SystemKustomizationPath = "kustomization.yaml"
 	WegoAppPath             = "wego-app.yaml"
 	WegoConfigPath          = "wego-config.yaml"
+	WegoProfilePath         = "profiles.yaml"
```

Comment:
I know we paired on this but we should see if we can remove having to have this here and in `git/git.go`. I think this is probably the better place